### ISSUE TITLE
Loading problems - attendance and demographics

### ISSUE THREAD

**Issue by gwenrr** [DC-OSSE/LearnDC_v2#161]
Some loading problems on the site - equity and profile both, PCS and DCPS both
![screenshot 2015-08-25 08 56 27](https://cloud.githubusercontent.com/assets/7264714/9467134/8477d3ce-4b07-11e5-85c6-9dc5ced32f38.png)
![screenshot 2015-08-25 08 56 11](https://cloud.githubusercontent.com/assets/7264714/9467137/8893d624-4b07-11e5-93c5-bb0be7ed3488.png)

**Comment by cmgiven** [comment]
Example school that shows the issue: http://www.learndc.org/schoolprofiles/view?s=3068#equityreport

**Comment by amberleyromo** [comment]
Looping in @psteininger to take look.

**Comment by psteininger** [comment]
@cmgiven I will take a look tomorrow morning. There seems to be some javascript errors. I'll update tomorrow

**Comment by michaelaskew-sd** [comment]
Separated the second screenshot into #162. This also appears to be specific to this school. We are trying to replicate locally and will test across other schools.

**Comment by psteininger** [comment]
@cmgiven @amberleyromo  - @michaelaskew-sd  and i spent some time debugging, and it turned out that the source of the problem was corrupted cache. For some reason some data was simply not coming back (i.e. year, which made impossible to get the max year for data, and subsequent call).

I checked the metrics on RedisCloud and it appears that what may very likely be the cause of corrupted cache is the fact that we ran out of the allotted 100MB. This happened probably because there were too many revisions active in the system. I deleted old revisions, and the cache usage went down to about 7MB.

We can think of something we can put in place to prevent running out of cache memory.
